russian/glue/wallpapers.md

```diff
@@ -0,0 +1,9 @@
+Да здесь всё так липко, и мне не соствит труда этого сделать.
+С чего начать? Ах да, мне же понадобятся некоторые инструменты да и обои.
+Какие бы мне выбрать?
+[эти](https://www.google.be/search?q=windows&tbm=isch) или может вот [эти](https://www.google.be/search?q=linux&tbm=isch)
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.